frog/imports/client/SingleActivity/types.js

```diff
@@ -10,6 +10,12 @@ export type StateT = {
   slug?: string
 };
 
+export type ShowModalFunctionT = (View: React.Node) => void;
+
+export type HideModalFunctionT = () => void;
```

Comment:
Should be removed, #1787 provide types for this

Reply:
Addressed